ports/zephyr/machine_wdt.c

```diff
@@ -0,0 +1,95 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 Paul Sokolovsky
+ * Copyright (c) 2017 Eric Poulsen
+ * Copyright (c) 2025 Daniel Campora on behalf of REMOTE TECH LTD
```

Comment:
I appreciate that you want to retain authorship here.  But this file is very simple and it doesn't look like there's much code actually copied from other sources, most of the code here is original work made to integrate with Zephyr.

How much was actually copied verbatim and unchanged?

We should definitely keep authorship when relevant, but it doesn't really make sense to unnecessarily keep copyright lines.